SourceKitStressTester/Sources/StressTester/Action.swift

```diff
@@ -12,7 +12,7 @@
 
 public enum Action: Equatable {
   case cursorInfo(offset: Int)
-  case codeComplete(offset: Int)
+  case codeComplete(offset: Int, expectedResult: ExpectedResult?)
```

Comment:
Not so important but I think it would be good to add documentation what a nil expected result means. I assume it simply doesn’t check the results but might be good to document.